Assets/HoloToolkit/Common/Scripts/Extensions/LayerExtensions.cs

```diff
@@ -54,15 +53,16 @@ public static int Activation
         /// <param name="layer">Layer to search for</param>
         /// <param name="layerMasks">List of LayerMasks to search</param>
         /// <returns>LayerMaskList index, or -1 for not found</returns>
-        public static int FindLayerListIndex(this int layer, IEnumerable<LayerMask> layerMasks)
+        public static int FindLayerListIndex(this int layer, LayerMask[] layerMasks)
```

Comment:
Changing the parameters of a method would be a breaking change, right? Is this change worth that?

Reply:
No, it's not a breaking change.

Reply:
Yes, this change is very worth it, considering it removes _all_ allocations for this check.

Reply:
It's a public method though, so devs might call it in their apps too.

What/how much do we gain from removing all allocations?